packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/add-ip-instance/controller.js

```diff
@@ -0,0 +1,73 @@
+export default class OctaviaLoadBalancerPoolsDetailMembersAddIpInstanceCtrl {
+  /* @ngInject */
+
+  constructor(Alerter, OctaviaLoadBalancerMembersService, $translate) {
+    this.Alerter = Alerter;
+    this.OctaviaLoadBalancerMembersService = OctaviaLoadBalancerMembersService;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.model = {
+      checked: [],
+    };
+    this.instances = [];
+    this.displayedInstances = [];
+    this.resetCurrentStep();
+    this.loadIpInstances();
+  }
+
+  loadIpInstances() {
+    this.isLoading = true;
+    this.OctaviaLoadBalancerMembersService.getInstances(this.projectId)
+      .then((instances) => {
+        angular.forEach(instances, (instance, index) => {
+          this.computeIpV4Instance(instance);
+          this.model.checked[index] = false;
+        });
+        this.displayedInstances = [...this.instances];
+      })
+      .catch((error) => {
+        this.Alerter.error(
+          this.$translate.instant('octavia_load_balancer_global_error', {
+            message: error.data?.message,
+            requestId: error.headers('X-Ovh-Queryid'),
+          }),
+          'octavia.alerts.members',
+        );
+      })
+      .finally(() => {
+        this.isLoading = false;
+      });
+  }
+
+  computeIpV4Instance(instance) {
+    instance.ipAddresses
+      .filter((ipAddress) => ipAddress.version === 4)
+      .forEach((ipAddress) => {
+        this.instances.push({
+          label: `${instance.name} (${ipAddress.ip}) `,
+        });
+      });
+  }
+
+  resetCurrentStep() {
+    this.currentStep = 0;
+  }
+
+  onSearchInstance(value) {
+    this.displayedInstances = this.instances.filter((instance) =>
+      instance.label.includes(value.toLowerCase()),
```

Comment:
I think you should also use toLowerCase on instance.label
```suggestion
      instance.label.toLowerCase().includes(value.toLowerCase()),
```

Reply:
Yes done.